editor.jsx

```diff
@@ -14,10 +20,61 @@ MediumEditorComp = React.createClass({
       paste: this.props.paste,
       keyboardCommands: this.props.keyboardCommands
     });
+
+    //Subscribe to changes and if there is a callback call it
+    instance.medium.subscribe('editableInput', function (event, editor) {
+      if(typeof instance.props.onChange === 'function') {
+        instance.props.onChange(event, instance.refs.editor.innerHTML);
+      }
+    });
   },
+
+  //Allow us to toggle the medium editor
+  toggleMediumEditor () {
+    if(this.props.edit) {
+      this.initMedium();
+    } else if(this.medium) {
+      this.medium.destroy();
+    }
+  },
+
+  //On mount toggle the editor based on the edit prop
+  componentDidMount() {
+    this.toggleMediumEditor();
+  },
+
+  //When the component is unmounted destroy the
+  //Non-isomorphic component
+  componentWillUnmount () {
+    if(this.medium)
+      this.medium.destroy();
+  },
+
+  //Check if we should update
+  shouldComponentUpdate (nextProps, nextState) {
+    return nextProps.edit !== this.props.edit;
+  },
+
+  //Toggle the editor when something updates
+  componentDidUpdate (prevProps, prevState) {
+      this.toggleMediumEditor();
+  },
+
+  //Render our component
   render() {
+    let instance = this;
     return(
-      <div className="editable" contentEditable="true" spellCheck="true" role="textbox" aria-multiline="true"></div>
+      <div
+        ref="editor"
+        className="editable"
+        contentEditable="true"
+        spellCheck="true"
+        role="textbox"
+        aria-multiline="true"
+        dangerouslySetInnerHTML={ { __html : instance.props.defaultValue } }
+        style={ this.props.style }
+        {...this.props.other}
```

Comment:
This doesn't seem to follow the same spec here: https://facebook.github.io/react/docs/transferring-props.html. Maybe I'm misunderstanding but it seems like you would need to define the '...other' as a variable else it'll contain all the things passed through?